delta/kernel/src/main/scala/ch/epfl/bluebrain/nexus/delta/kernel/utils/ClasspathResourceUtils.scala

```diff
@@ -14,7 +14,9 @@ import scala.jdk.CollectionConverters._
 
 trait ClasspathResourceUtils {
 
-  final def absolutePath(resourcePath: String)(implicit classLoader: ClassLoader): IO[String] =
+  private val classLoader: ClassLoader = getClass.getClassLoader
+
+  final def absolutePath(resourcePath: String): IO[String] =
```

Comment:
Removing the classloader is very likely to cause some problems when loading some files in the plugins.
It is a good opportunity to see if it is really mandatory though :)

Reply:
This was an issue, the fix is pushed